.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Notify @EWDurbin for all opened Issues and Pull Requests
```

Comment:
Name doesn't match list, no need to repeat names in comment, and it'll only assign for PRs (that touch code), not issues:
```suggestion
# Notify for all opened pull requests
```